me.eastsun.hackdesk.yml

```diff
@@ -0,0 +1,24 @@
+app-id: me.eastsun.hackdesk
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: hackdesk
+
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=wayland # Permission needed to show the window
+  - --socket=x11 # Permission needed to show the window
+  - --socket=fallback-x11
+  - --filesystem=home/.hackdesk:create # Permission needed to save the config
```

Comment:
no. either conform to the xdg standard or use `--persist`

Reply:
Fixed!